models/CommentManager.php

```diff
@@ -0,0 +1,131 @@
+<?php
+// Permet d'avoir accès à la base de données
+
+class CommentManager extends Model{
+    
+    //FONCTIONS POUR LE FRONTEND
+    // Fonction qui permet de récupérer les commentaires visibles d'un article (seen = 1)
+     function get_comments_blog(){
+        $req = $this->db->query("SELECT * FROM comments WHERE article_id = '{$_GET['id']}' AND seen = 1 ORDER BY date DESC");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized, non-validated input variable $_GET: "SELECT * FROM comments WHERE article_id = '{$_GET['id']}' AND seen = 1 ORDER BY date DESC"](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)